feeder/cmd/pixel_bt_feeder/main.go

```diff
@@ -0,0 +1,261 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// pixel_bt_feeder is an implementation of a witness feeder for the Pixel BT log.
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/binary"
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+	"time"
+	"unicode"
+	"unicode/utf8"
+
+	"github.com/golang/glog"
+	"github.com/google/trillian-examples/formats/log"
+	"github.com/google/trillian-examples/internal/feeder"
+	"github.com/google/trillian-examples/serverless/config"
+	"golang.org/x/mod/sumdb/note"
+	"golang.org/x/mod/sumdb/tlog"
+
+	i_note "github.com/google/trillian-examples/internal/note"
+	wit_http "github.com/google/trillian-examples/witness/golang/client/http"
+	yaml "gopkg.in/yaml.v2"
+)
+
+const (
+	// tileHeight is the tlog tile height.
+	// From: https://developers.devsite.corp.google.com/android/binary_transparency/tile
+	tileHeight = 1
```

Comment:
tileHeightBytes?

Reply:
I think it's literally the depth of tile tree - so Pixel BT has 2 hashes per tile...